packages/workflow/src/Workflow.ts

```diff
@@ -140,12 +137,30 @@ export class Workflow {
 		this.staticData.__dataChanged = true;
 	}
 
+	/** This method removes any ghost connections/nodes */
+	getConnectionsBySourceNode(connections: IConnections) {
+		const returnConnection: IConnections = {};
+		for (const sourceNode in connections) {
+			if (sourceNode in this.nodes) {
+				returnConnection[sourceNode] = connections[sourceNode];
+				for (const inputConnections of Object.values(returnConnection[sourceNode])) {
+					inputConnections.forEach((innerConnections, index) => {
+						if (innerConnections?.length) {
+							inputConnections[index] = innerConnections?.filter(({ node }) => node in this.nodes);
```

Comment:
```suggestion
							inputConnections[index] = innerConnections.filter(({ node }) => node in this.nodes);
```

Already proved above.